[2024-11-24T06:14:15.007+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model permission to 509
[2024-11-24T06:14:15.007+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model'
[2024-11-24T06:14:15.034+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model permission to 509
[2024-11-24T06:14:15.035+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model'
[2024-11-24T06:14:15.057+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_training_pipeline.train_model manual__2024-11-24T06:09:52.679856+00:00 [queued]>
[2024-11-24T06:14:15.065+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_training_pipeline.train_model manual__2024-11-24T06:09:52.679856+00:00 [queued]>
[2024-11-24T06:14:15.065+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-11-24T06:14:15.082+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): train_model> on 2024-11-24 06:09:52.679856+00:00
[2024-11-24T06:14:15.093+0000] {standard_task_runner.py:57} INFO - Started process 20405 to run task
[2024-11-24T06:14:15.095+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'model_training_pipeline', 'train_model', 'manual__2024-11-24T06:09:52.679856+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/model_training_pipeline_dag/model_training_dag.py', '--cfg-path', '/tmp/tmpeq295f9e']
[2024-11-24T06:14:15.097+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask train_model
[2024-11-24T06:14:15.138+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model permission to 509
[2024-11-24T06:14:15.138+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=model_training_pipeline/run_id=manual__2024-11-24T06:09:52.679856+00:00/task_id=train_model'
[2024-11-24T06:14:15.140+0000] {task_command.py:415} INFO - Running <TaskInstance: model_training_pipeline.train_model manual__2024-11-24T06:09:52.679856+00:00 [running]> on host 71596b931411
[2024-11-24T06:14:15.208+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='model_training_pipeline' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2024-11-24T06:09:52.679856+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-24T06:09:52.679856+00:00'
[2024-11-24T06:17:32.082+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-11-24T06:17:32.085+0000] {process_utils.py:131} INFO - Sending 15 to group 20405. PIDs of all processes in the group: [20405]
[2024-11-24T06:17:32.085+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 20405
[2024-11-24T06:17:32.086+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-11-24T06:17:32.138+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=20405, status='terminated', exitcode=0, started='06:14:14') (20405) terminated with exit code 0
